### PR TITLE
Ebuild timeplot

### PR DESCRIPTION
**Pull request by vespos**
Add the option to modify the time span of the ebuild timeplots.

Tested with ebuild IOC at XPP.

### CODE REVIEW

File: widgets/event_builder_timeplot.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+TICK_FONT = QFont()
+TICK_FONT.setPixelSize(16)
+
+class EventBuilderTimeplot(Display):
+    def __init__(self, parent=None, args=None, macros=None):
+        """ 
+        Screen to plot time plots based of the ebuild waveform PV. Allows adjustement of 
+        the timespan via a QLineEdit.
+        """
+        super(EventBuilderTimeplot, self).__init__(parent=parent, args=args, macros=macros)
```
Comment (ZLLentz) [inline widgets/event_builder_timeplot.py:21]: In Python 3 you can simplify this to
```
super().__init__(parent=parent, args=args, macros=macros)
```

**Review by ZLLentz — APPROVED** [review]
This contains the previous PR inside of it but otherwise looks good
Left some optional suggestions

File: widgets/event_builder_timeplot.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        return 'event_builder_timeplot.ui'
+
+
+    def ui_filepath(self):
+        return path.join(path.dirname(path.realpath(__file__)), self.ui_filename())
+
+
+    @Slot()
+    def update_timespan(self):
+        timespan = int(self.le_timespan.text())
```
[the remark this answers is not in the document]
Comment (vespos) [reply widgets/event_builder_timeplot.py:30]: I addedd the IntValidator to the LineEdit, no it's impossible to type anything else but an integer

**Comment by vespos** [comment]
Oh yes, I branched off the other branch instead of master... my bad.

**Comment by ZLLentz** [comment]
It's not a big deal, if we merge the other one first then this one's diff should update appropriately

File: widgets/event_builder_timeplot.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ZLLentz) [reply widgets/event_builder_timeplot.py:32]: Solid solution, I missed that in my read

**Pull request merged**